### DISCUSSION TITLE
Unicode is not supported

### DISCUSSION THREAD

**Discussion by Cossack13** [selectize/selectize.js#1925] [Q&A]
The search for Russian characters does not work.

**Comment by Cossack13** [comment]
This worked correctly in version v0.12.6
https://snipp.ru/jquery/select-search

**Comment by ttskch** [comment]
@Cossack13 

Maybe this is caused from `respect_word_boundaries` option.
From v0.15.0, `respect_word_boundaries` option is turned to true by default.

> refs:
> 
> * https://github.com/selectize/selectize.js/blob/v0.15.0/src/defaults.js#L47
> * https://github.com/selectize/selectize.js/commit/ca965de07b36aecd44782e5f2638cfb98303659a#diff-8fd0ad3fc60cdee531fcad524733fcabe2fd5f86758a5f1c4cee4d87aea19b78R47

This causes a known problem.
https://github.com/selectize/selectize.js/issues/1916

For now, you may can resolve this problem with like following.

```js
$('#target').selectize({  
  respect_word_boundaries: false,  
})
```
